src/Symfony/Component/Form/Resources/translations/validators.ro.xlf

```diff
@@ -1,19 +1,139 @@
 <?xml version="1.0"?>
 <xliff version="1.2" xmlns="urn:oasis:names:tc:xliff:document:1.2">
-    <file source-language="en" datatype="plaintext" original="file.ext">
-        <body>
-            <trans-unit id="28">
-                <source>This form should not contain extra fields.</source>
-                <target>Aceast formular nu ar trebui să conțină câmpuri suplimentare.</target>
-            </trans-unit>
-            <trans-unit id="29">
-                <source>The uploaded file was too large. Please try to upload a smaller file.</source>
-                <target>Fișierul încărcat a fost prea mare. Vă rugăm sa încărcați un fișier mai mic.</target>
-            </trans-unit>
-            <trans-unit id="30">
-                <source>The CSRF token is invalid. Please try to resubmit the form.</source>
-                <target>Token-ul CSRF este invalid. Vă rugăm să trimiteți formularul incă o dată.</target>
-            </trans-unit>
-        </body>
-    </file>
-</xliff>
\ No newline at end of file
+	<file datatype="plaintext" original="file.ext" source-language="en">
+		<body>
+			<trans-unit id="28">
+				<source>This form should not contain extra fields.</source>
+				<target>Aceast formular nu ar trebui să conțină câmpuri suplimentare.</target>
+			</trans-unit>
+			<trans-unit id="29">
+				<source>The uploaded file was too large. Please try to upload a smaller file.</source>
+				<target>Fișierul încărcat a fost prea mare. Vă rugăm sa încărcați un fișier mai mic.</target>
+			</trans-unit>
+			<trans-unit id="30">
+				<source>The CSRF token is invalid. Please try to resubmit the form.</source>
+				<target>Token-ul CSRF este invalid. Vă rugăm să trimiteți formularul incă o dată.</target>
```

Comment:
"încă" instead of "incă" is the only fix I think the translation needs

Reply:
IMHO, a better translation for this could be the following: `Token-ul CSRF este invalid. Vă rugăm să retrimiteți formularul.`, thinking from users perspective.